src/System.Net.NetworkInformation/src/project.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "System.Diagnostics.Contracts": "4.0.0-*",
+    "System.Threading": "4.0.0-*",
+    "System.Net.Primitives": "4.0.10-beta-*",
+    "System.Net.Sockets": "4.0.10-beta-*"
+  },
+  "frameworks": {
+    "dnxcore50": {}
+  }
+}
```

Comment:
We should make sure we're using the stable contract versions wherever we can (we may already be).